README.rst

```diff
@@ -100,3 +100,39 @@ API
   **default** - The value to return if the environment variable is not present.
   (providing a default alongside setting ``required=True`` will raise a
   ``ValueError``)
+
+* ``excavator.env_timestamp(name, required=False, default='', format='timestamp')``::
```

Comment:
Docs need to be updated with the new function signatures for timestamp and iso8601.